src/hanoi-tower.js

```diff
@@ -1,6 +1,9 @@
 const CustomError = require("../extensions/custom-error");
 
-module.exports = function calculateHanoi(/* disksNumber, turnsSpeed */) {
-  throw new CustomError('Not implemented');
-  // remove line with error and write your code here
+module.exports = function calculateHanoi(disksNumber, turnsSpeed) {
+  let turns = 2**disksNumber - 1;
+  let seconds = 3600/turnsSpeed*turns;
+  return {'turns': turns, 'seconds': Math.floor(seconds)};
```

Comment:
you don't have to put key into quotes. also if you key has same name as value variable, you can merge them. like this:

return {turns, seconds} (if you move Math.floor to line 5